code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -407,10 +407,6 @@
 	glass_name = "glass of Nuka Cola"
 	glass_desc = "Don't cry, Don't raise your eye, It's only nuclear wasteland."
 
-/datum/reagent/consumable/nuka_cola/on_mob_metabolize(mob/living/L)
-	..()
-	L.add_movespeed_modifier(id, update=TRUE, priority=100, multiplicative_slowdown=-1, blacklisted_movetypes=(FLYING|FLOATING))
```

Comment:
Craplord 10k milestone PR. I don't give a heck about the fact ke:b:in removed the gotta go fast from meth and ephedrine, that was before those traits were converted into a movespeed modifier, which are indefinitely better and customizable. Just reduce the slowndown value to minus 0.75 or 0.5 or something.